prov/util/src/util_coll.c

```diff
@@ -764,6 +747,33 @@ static struct fi_ops util_coll_fi_ops = {
 	.ops_open = fi_no_ops_open,
 };
 
+static inline int util_coll_mc_init(struct util_coll_mc *coll_mc,
+				   struct util_av_set *av_set,
+				   struct fid_ep *ep, void *context)
+{
+	coll_mc->mc_fid.fid.fclass = FI_CLASS_MC;
+	coll_mc->mc_fid.fid.context = context;
+	coll_mc->mc_fid.fid.ops = &util_coll_fi_ops;
+	coll_mc->mc_fid.fi_addr = (uintptr_t) coll_mc;
+	coll_mc->ep = ep;
+	assert(av_set != NULL);
+	ofi_atomic_inc32(&av_set->ref);
+	coll_mc->av_set = av_set;
+
+	return FI_SUCCESS;
+}
+
+int ofi_av_set_addr(struct fid_av_set *set, fi_addr_t *coll_addr)
```

Comment:
This can be static.

Reply:
This is an API function implementation. Should it be static? I see other ofi_ functions are not static, e.g. ofi_join_collective.

Reply:
If it's not used outside of this file, I would make it static.  It's exposed through a function pointer.  ofi_join_collective is accessed by a separate file.